writers/mixedbatchwriter/mixedbatchwriter.go

```diff
@@ -218,11 +220,16 @@ func (m *insertBatchManager) flush(ctx context.Context) error {
 	if len(m.batch) == 0 {
 		return nil
 	}
-
+	start := time.Now()
+	batchSize := len(m.batch)
```

Comment:
You can move this to line 220

Reply:
but this is more readable I guess, and shouldn't matter if the compiler does its job